code/src/sixsq/nuvla/server/resources/voucher.clj

```diff
@@ -0,0 +1,224 @@
+(ns sixsq.nuvla.server.resources.voucher
+  "
+This resource contains the structure for a voucher, which
+is to be issued by a third party and used by any Nuvla user.
+"
```

Comment:
It would be helpful is the description also contained information about the workflow/state machine of the resource.

Reply:
done